src/pages/RedeemHoloFuelPage.vue

```diff
@@ -29,13 +34,25 @@ async function getRedeemableHoloFuel(): Promise<void> {
   isLoading.value = false
 }
 
-const redeemableHoloFuel = computed(() => {
-  if (isError(dashboardStore.hostEarnings)) {
-    return dashboardStore.hostEarnings
-  }
+interface ExtendedRedemptionTransaction extends RedemptionTransaction {
+  hotAddress: string
+}
 
-  return dashboardStore.hostEarnings.holofuel.redeemable || '0'
-})
+async function handleRedemptionSubmitted(
+  transaction: ExtendedRedemptionTransaction
+): Promise<void> {
+  await router.push({ name: kRoutes.earnings.name })
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-return
+  showModal(EModal.redemption_initiated, {
+    date: transaction.created_date,
+    hfAmount: transaction.amount,
+    requestId: transaction.id,
+    currency: 'HOT',
```

Comment:
Can we add a quick note explaining that this is only hard coded until we add the call to support other Reserve types (functionality already exists in back-end).